examples/todoapp/lib/l10n/arb/feature_deep_link/en.arb

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove all files related to not-used features

Reply:
Done